src/eradius.app.src

```diff
@@ -1,7 +1,7 @@
 %% vim: ft=erlang
 {application, eradius, [
    {description, "Erlang RADIUS server"},
-   {vsn, "0.8.1"},
```

Comment:
wrong version, there have been changes since 0.8.1 was released.

Reply:
Now it takes version from git tag. I'm going to bump it when this PR will be done